core/engine_test.go

```diff
@@ -535,7 +535,7 @@ func TestSentReceivedMetrics(t *testing.T) {
 		{tr(`import ws from "k6/ws";
 			let data = "0123456789".repeat(100);
 			export default function() {
-				ws.connect("WSBIN_URL/ws-echo", null, function (socket) {
+				ws.connect("WSBIN_URL/ws-echo-invalid", null, function (socket) {
```

Comment:
All of these changes are to indicate this is an "invalid" server implementation, which I left in to avoid refactoring all existing tests that depended on it. But let me know if you prefer a different approach.

Reply:
Wasn't just one test that was affected `TestSentReceivedMetrics` (this one) . Why are you using it for all the ws_tests as well ?

Reply:
I thought I had another test failure because of the change, so I decided to update them all and avoid any unexpected effects.

But I just tested and most tests work fine with the fixed server implementation, so I reverted these changes in 2ac9618b, add added a comment to this test explaining this, and a small `FIXME` note, let me know if you agree.